engine/runtime-compiler/src/main/scala/org/enso/compiler/context/InlineContext.scala

```diff
@@ -18,22 +18,21 @@ import org.enso.compiler.pass.PassConfiguration
   * @param pkgRepo the compiler's package repository
   */
 case class InlineContext(
-  private val module: ModuleContext,
+  moduleContext: ModuleContext,
```

Comment:
It was `private`. Why open up?

Reply:
Hmmm good question. I think you are right this is unnecessary.

I think I needed it exposed when experimenting with something and then forgot to hide it again after the experiments turned out a dead-end. Sorry for the oversight and thanks for pointing it out.